xml/obs_glossary.xml

```diff
@@ -34,6 +34,20 @@
    </para>
   </glossdef>
  </glossentry>
+ <glossentry xml:id="obs.glos.apidoc">
+  <glossterm>API</glossterm>
+  <glossdef>
+   <para>
+    API stands for application programming interface. It let your product or service communicate with other products and services without having to know how they’re implemented.
+   </para>
+   <para>
+    The OBS API is located here: <link xlink:href="https://api.opensuse.org" />.
+   </para>
+   <para>
+    The documentation for the API is located here: <link xlink:href="https://api.opensuse.org/apidocs" />.
+   </para>
```

Comment:
```suggestion
   <para>
    The OBS API is documented here: <link xlink:href="https://api.opensuse.org/apidocs/" />.
   </para>
```
Providing the direct link to the OBS API doesn't add as much value as providing the link to the documentation, IMHO. Suggesting both links sounds to me kind of redundant. What do you think?

Reply:
Well, the Glossary talks about the API, not the documentation. Placing a link to the API documentation out of the blue, it's kind of disconnected. So to connect the documentation with the API I need to mention the API first.

Reply:
Fair point. :+1:

Reply:
In my naive view, the main page of `api.opensuse.org` would contain all the necessary links. :wink:  That would make it easier for this glossary item as you only have to link to one link.

Reply:
> In my naive view, the main page of `api.opensuse.org` would contain all the necessary links. wink That would make it easier for this glossary item as you only have to link to one link.

Noted down. We track now your suggestion in one of our cards: https://trello.com/c/oWhGOnN4/2975-change-what-to-show-in-https-apiopensuseorg

 Thanks for the feedback. Please keep it comming!